packages/core/src/drawer/drawer.scss

```diff
@@ -0,0 +1,95 @@
+:host {
+  --_background: #fff;
+  --_border: 0;
+  --_border-radius: 3px;
+  --_box-shadow: 0 6px 10px -6px rgba(0, 0, 0, 0.7);
+  --_gap: 0.5rem;
+  --_spacing: 0.5rem;
+  --_max-inline-size: var(--sl-drawer-max-inline-size, 500px);
+}
+
+:host {
+  display: block;
+}
+
+dialog {
+  background: var(--_background);
+  border: var(--_border);
+  border-radius: var(--_border-radius);
+  box-shadow: var(--_box-shadow);
+  display: flex;
+  flex-direction: column;
+  gap: var(--_gap);
+  margin: 0;
+  max-block-size: min(100vh, 100%);
+  max-block-size: min(100dvb, 100%);
+  max-inline-size: min(90vw, var(--_max-inline-size));
+  padding: 1rem;
+  position: fixed;
+  transform: translate(var(--fade-start-x, 0), var(--fade-start-y, 0))
+    scale(var(--fade-start-sx, 1), var(--fade-start-sx, 1));
+
+  &::backdrop {
+    /* stylelint-disable-next-line property-no-vendor-prefix */
+    -webkit-backdrop-filter: blur(3px);
+    backdrop-filter: blur(3px);
+    transition: backdrop-filter 0.5s ease;
+  }
+}
+
+dialog[open] {
+  transform: translate(0, 0) scale(1, 1);
+}
+
+div {
+  align-items: center;
+  display: grid;
+  gap: var(--_gap);
+  grid-auto-flow: column;
```

Comment:
Yes, because of the grid-column-start for the close button, otherwise it would make 2 rows and the title and other action buttons would be on the second row